modules/ocf/manifests/packages/firefox.pp

```diff
@@ -3,14 +3,20 @@
     stage => first,
   }
 
-  $firefox_pkg = $::lsbdistcodename ? {
-    'jessie' => 'firefox',
-    default  => 'firefox-esr',
+  if $::lsbdistcodename == 'jessie' {
```

Comment:
I don't think you need the if statement here, `ocf::repackage` will install the regular way (using a `package` resource) if it doesn't match the distribution to "backport" on (not actually backporting in this case), so just having the `ocf::repackage` resource below should be enough.